Assets/Scripts/Utilities/WadeUtils.cs

```diff
@@ -384,16 +379,16 @@ public static GameObject TempInstantiate(GameObject obj, Vector3 pos, Quaternion
 	//////  FRAME CHECKING   ////////
 	/////////////////////////////////
 
-	public static bool IsWithinFrame(float currentFrame, int targetFrame)
+	public static bool IsWithinFrame(float currentFrame, int targetFrame) // this seems so jank
 	{
 		return Mathf.Abs(currentFrame - targetFrame) <= 1f;
```

Comment:
This looks like it has a two frame window. E.g. 5.0f and 3.0f would both be "within" frame 4. Is that intended?